torch/testing/_internal/common_utils.py

```diff
@@ -162,6 +163,40 @@ def def_flag(
         assert name not in globals(), f"duplicate definition of flag '{name}'"
         globals()[name] = enabled
 
+    # Defines a setting usable throughout the test suite, determining its value by querying
+    # the specified environment variable. This differs from a flag in that it's not restricted
+    # to a boolean value.
+    #
+    # Args:
+    #     name (str): The name of the setting. A global variable with this name will be set
+    #         for convenient access throughout the test suite.
+    #     env_var (str): The name of the primary environment variable from which to
+    #         determine the value of this setting. If this is None or the environment variable
+    #         is unset, the default value will be used. Default: None
+    #     default (bool): The default value to use for the setting if unset by the environment
```

Comment:
can't default be more than a bool here?

Reply:
yep you're right! copy / paste error from `def_flag()`- will fix